### PR TITLE
Add student box styles

### PR DESCRIPTION
**Pull request by afmonty**
From the Details page,  the add student pop-up styling

### CODE REVIEW

**Comment by NJHawthorne** [comment]
👍

**Comment by MHG16** [comment]
👍

**Comment by willcook4code** [comment]
👍

**Comment by alarner** [comment]
Please resolve merge conflicts

**Comment by afmonty** [comment]
this will need to get new thumbs... I forgot to change branches before I started working on this.

File: public/scripts/components/Navigation.js [hunk 1]
```diff
@@ -18,12 +20,41 @@ export default React.createClass({
...
+		Stories.on('update', () =>{
+			this.setState({
+				Stories: Stories,
+				//categories: categories
+			});
+		});	
 	},
 
 	render: function() {
+<<<<<<< HEAD
```
Comment (alarner) [inline public/scripts/components/Navigation.js]: There's some merge conflict code here

File: public/scripts/components/Navigation.js [hunk 2]
```diff
@@ -18,12 +20,41 @@ export default React.createClass({
...
+		if(this.state.user.get('id')) {
+			return (<nav>
+			<img src="/images/pointer_logo.png" />
+			<div className="nav-cat-dropdown">	
+				<select value = "Select a category" name="mydropdown" onChange={this.selectCat}>
+					<option value="Pick">Go to Category</option>
+							{allCategories}
+				</select>
+			</div>
+=======
```
Comment (alarner) [inline public/scripts/components/Navigation.js]: and here

**Comment by alarner** [comment]
There is some merge conflict code.

**Comment by alarner** [comment]
👍

**Pull request merged**